src/renderer/components/ft-list-lazy-wrapper/ft-list-lazy-wrapper.js

```diff
@@ -36,11 +36,46 @@ export default Vue.extend({
   computed: {
     hideLiveStreams: function() {
       return this.$store.getters.getHideLiveStreams
+    },
+    channelsHidden: function() {
+      return JSON.parse(this.$store.getters.getChannelsHidden)
     }
   },
   methods: {
     onVisibilityChanged: function (visible) {
       this.visible = visible
+    },
+
+    /**
+     *  Show or Hide results in the list
+     *
+     * @return {bool} false to hide the video, true to show it
+     */
+    showResult: function (data) {
+      if (data.type !== undefined) {
+        if (data.type === 'video') {
+          if ((data.liveNow || data.lengthSeconds == null) && this.hideLiveStreams) {
+            // hide livestreams
+            return false
+          }
+          if (this.channelsHidden.includes(data.authorId) || this.channelsHidden.includes(data.author)) {
+            // hide videos by author
+            return false
+          }
+        } else if (data.type === 'channel') {
+          if (this.channelsHidden.includes(data.channelID) || this.channelsHidden.includes(data.name)) {
+            // hide channels by author
+            return false
+          }
+        } else if (data.type === 'playlist') {
+          if (this.channelsHidden.includes(data.authorId) || this.channelsHidden.includes(data.author)) {
+            // hide playlists by author
+            return false
+          }
+        }
+      }
+      return true
```

Comment:
the previous check hid the item if `data` is undefined, this implementation doesn't

Reply:
I believe the conflict resolutions in the rebase below will resolve this as well. See [L59](https://github.com/FreeTubeApp/FreeTube/commit/4d618e3b5459d72eafb9a98fca5c4a9b7a3d1182#diff-73b4b93ba9f89c56a9a2517a9ceb144b6fb10a00bdecb8c70b1291e4a77827b0R59).

Reply:
Yes this is fixed with the merge with latest development